osu.Game/Screens/Edit/GameplayTest/EditorPlayer.cs

```diff
@@ -164,7 +164,7 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
                 case GlobalAction.EditorTestPlayQuickExitToCurrentTime:
                     quickExit(true);
                     return true;
-                    
+
```

Comment:
you need to fix your editing environment and not make commits like this. there's a similar whitespace issue in your latest commits too.